mousestyles/classification/Classification.py

```diff
@@ -0,0 +1,72 @@
+from mousestyles import data
+import numpy as np
+from sklearn.grid_search import GridSearchCV
+from sklearn.ensemble import RandomForestClassifier
+from sklearn.preprocessing import StandardScaler
+import pandas as pd 
+
+def RandomForest():
```

Comment:
Here, you need to add doc string for the function, specify what the function does, what parameters it take, what output it returns, provide examples etc